source/autocomplete/autocomplete_api.cpp

```diff
@@ -40,49 +40,49 @@ static void create_place_pb(const std::vector<Autocomplete<nt::idx_t>::fl_qualit
                             const nt::Type_e type,
                             uint32_t depth,
                             const nt::Data& data,
-                            pbnavitia::Response& pb_response){
+                            navitia::PbCreator& pb_creator){
     for(auto result_item : result){
-        pbnavitia::PtObject* place = pb_response.add_places();
+        pbnavitia::PtObject* place = pb_creator.add_places();
         switch(type){
         case nt::Type_e::StopArea:
-            navitia::fill_pb_object(data.pt_data->stop_areas[result_item.idx], data, place, depth);
+            pb_creator.fill(data.pt_data->stop_areas[result_item.idx], place, depth);
```

Comment:
depth should not be needed here. A small cleanup of repassing depth can be done I think.